src/theme.tsx

```diff
@@ -1,95 +1,111 @@
-import { createTheme, responsiveFontSizes } from '@mui/material'
+import { createTheme } from '@mui/material'
 
-export const kurtTheme = responsiveFontSizes(
-  createTheme({
-    typography: {
-      allVariants: {
-        font: 'Roboto',
-      },
-      fontFamily: ['Roboto', 'sans-serif'].join(','),
-      h1: {
-        fontSize: '28px',
-        color: '#FFFFFF',
-        fontWeight: 700,
-      },
-      h2: {
-        fontWeight: 700,
-        color: '#FFFFFF',
-        fontSize: '24px',
-      },
-      body1: {
-        color: 'white',
-        textDecoration: 'none',
-        lineHeight: '18.75px',
-        fontSize: '16px',
-        fontWeight: 400,
-      },
-      body2: {
-        color: '#212529',
-        fontWeight: 400,
-        fontSize: '12px',
-        lineHeight: '14px',
-      },
-      button: {
-        // EXPLORE THE SERIES
-        color: '#FFFFFF',
-        lineHeight: '22px',
-        fontSize: '13px',
-        letterSpacing: '0.46px',
-      },
+export const kurtTheme = createTheme({
+  typography: {
+    allVariants: {
+      font: 'Roboto',
     },
-    palette: {
-      director_producer: {
-        main: '#9EC5FE',
-      },
-      director_of_photography: {
-        main: '#9EEAF9',
-      },
-      series: { main: '#F7D6E6' },
-      animation: { main: '#A3CFBB' },
-      documentary: { main: '#FECBA1' },
-      background: {
-        default: '#343A40',
+    fontFamily: ['Roboto', 'sans-serif'].join(','),
+    h1: {
+      fontSize: '1.75rem',
```

Comment:
Updated fontSizes to `rem`. We should use the fontSize in Figma  divided by the root html font size (current 16)